priv/graphql/schemas/user/account.gql

```diff
@@ -0,0 +1,16 @@
+"""
+Allow user to get information about account.
```

Comment:
Note: Here it should be "Allow a user to get information about their account", but for me there is no need to correct it as in comments one can skip articles.